### PR TITLE
Upgraded phpunit to allow 5.7 and 6.0

### PR DESCRIPTION
**Pull request by cmodijk**

### CODE REVIEW

**Comment by garak** [comment]
We should also test against lowest versions. If you try this with `composer update --prefer-lowest`, tests will fail. A possible solution is dropping support for Symfony 2.3.

Error is: `PHP Fatal error:  Class 'Symfony\Bundle\FrameworkBundle\Test\KernelTestCase' not found in /SymfonyBridge/tests/Functional/SmokeTest.php on line 14`

**Comment by cmodijk** [comment]
@garak I created a new issue for the drop of symfony 2.3 and lower https://github.com/SimpleBus/SymfonyBridge/issues/51

File: composer.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
         "symfony/monolog-bundle": "For logging messages"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.0 || ~5.0",
+        "phpunit/phpunit": "^4.0 || ^5.7 || ^6.0",
```
Comment (garak) [inline composer.json:35]: Is phpunit 4.0 still needed?

**Comment by ruudk** [comment]
Handled in #63

**Pull request closed without merging**